ios/browser/api/ads/brave_ads.mm

```diff
@@ -266,6 +244,12 @@ - (void)initialize:(void (^)(bool))completion {
   adsClient = new AdsClientIOS(self);
   adsClientNotifier = new brave_ads::AdsClientNotifier();
   ads = brave_ads::Ads::CreateInstance(adsClient);
+  ads->SetSysInfo(brave_ads::mojom::SysInfo::New(
+      base::SysNSStringToUTF8(sysInfo.deviceId)));
+  ads->SetBuildChannel(brave_ads::mojom::BuildChannelInfo::New(
+      buildChannelInfo.isRelease,
+      base::SysNSStringToUTF8(buildChannelInfo.name)));
+  ads->SetFlags(brave_ads::BuildFlags());
```

Comment:
Don't we need to be able to pass this in as well for iOS?

Reply:
Never mind, realize this `brave_ads::BuildFlags` is going to pull args from command line by default